Docker.md

```diff
@@ -0,0 +1,121 @@
+| /usr/local/tomcat/conf/server.xml | mount/copy of your server.xml | Contains the default server.xml of Tomcat, replace to secure your application |
+| /usr/local/tomcat/conf/catalina.properties | Server properties, contains default framework values | Do not replace this file, use [Environment variables](#Environment-variables) or append to the file, see [Dockerfile](docker/appserver/Tomcat/Dockerfile) for an example |
+
+## Permissions
+
+As this image does not run Tomcat using the root user, file permissions need to be set correctly. By default this is done during startup, to ensure that all files copied to the above locations have the correct permissions. For images with a large number of files in these locations, this can reduce startup performance. It is possible to set the permissions during build and disable the step during startup by adding the following lines at the end of your Dockerfile:
+```
+RUN /setPermissions.sh
+ENV SET_PERMISSIONS_ON_STARTUP=FALSE
+```
+
+Logging
+=======
+
+Generated log files are stored in `/usr/local/tomcat/logs`.
+
+Environment variables
+=====================
+
+Environment variables can be used to set parameters. Environment variables have the highest precedence and override parameters set in .property files supplied by Tomcat, resources and configurations.
+
+As `org.apache.tomcat.util.digester.PROPERTY_SOURCE=org.apache.tomcat.util.digester.EnvironmentPropertySource` is set in our images, environment variables can also be used to replace parameters in Tomcat configuration files such as server.xml and context.xml.
+
+Health and readiness
+====================
+
+The health and readiness of the container can be monitored by polling the `/iaf/api/server/health` API endpoint. This will return a HTTP statuscode of 200 if all adapters are running and a HTTP statuscode of 503 if there are adapters in a non-running state.
+
+Considerations
+==============
+
+The images are based on Tomcat, all restrictions and considerations that apply to Tomcat also apply to using the provided images.
+
+## HTTPS and authentication
+
+By default, the image uses the default server.xml of Tomcat which is not configured for inbound HTTPS traffic and user authentication on the administration console. To secure your application, replace server.xml with a secured version matching your requirements.
+
+Note: The GUI will not load data if accessed via HTTP and `dtap.stage!=LOC`, HTTP-based listeners will still process messages.
+
+## Secrets
+
+Special consideration should be taken with secrets. As described on the [Tomcat website](https://cwiki.apache.org/confluence/display/TOMCAT/Password), passwords are stored in plain text. To use secrets in your Tomcat and Frank!Application configuration, you can take the following steps:
+- In your configuration, refer to the username as `${<secret-name>/username}` and password as `${<secret-name>/password}`
```

Comment:
- In Frank!Application configuration, use `authAlias` to refer to a combination of username and password

Reply:
Misschien goed om dat in IAF-TEST voor Tomcat ook volledig in te richten, nu wordt [ConfigurationAuthentication](https://github.com/ibissource/iaf/blob/master/test/src/main/configurations/MainConfig/ConfigurationAuthentication.xml) niet actief gezet voor Tomcat, omdat de betreffende authAlias op Tomcat niet wordt ingericht (is nog uit de tijd dat we geen credentialProvider hadden)